crates/bridge_freetype2/src/lib.rs

```diff
@@ -1,5 +1,481 @@
+    pub isFixedPitch: libc::c_ulong,
+    pub minMemType42: libc::c_ulong,
+    pub maxMemType42: libc::c_ulong,
+    pub minMemType1: libc::c_ulong,
+    pub maxMemType1: libc::c_ulong,
+}
+
+#[repr(C)]
+pub struct TT_OS2 {
+    pub version: libc::c_ushort,
+    pub xAvgCharWidth: libc::c_short,
+    pub usWeightClass: libc::c_ushort,
+    pub usWidthClass: libc::c_ushort,
+    pub fsType: libc::c_ushort,
+    pub ySubscriptXSize: libc::c_short,
+    pub ySubscriptYSize: libc::c_short,
+    pub ySubscriptXOffset: libc::c_short,
+    pub ySubscriptYOffset: libc::c_short,
+    pub ySuperscriptXSize: libc::c_short,
+    pub ySuperscriptYSize: libc::c_short,
+    pub ySuperscriptXOffset: libc::c_short,
+    pub ySuperscriptYOffset: libc::c_short,
+    pub yStrikeoutSize: libc::c_short,
+    pub yStrikeoutPosition: libc::c_short,
+    pub sFamilyClass: libc::c_short,
+
+    pub panose: [libc::c_uchar; 10],
+
+    pub ulUnicodeRange1: libc::c_ulong,
+    pub ulUnicodeRange2: libc::c_ulong,
+    pub ulUnicodeRange3: libc::c_ulong,
+    pub ulUnicodeRange4: libc::c_ulong,
+
+    pub achVendID: [libc::c_char; 4],
+
+    pub fsSelection: libc::c_ushort,
+    pub usFirstCharIndex: libc::c_ushort,
+    pub usLastCharIndex: libc::c_ushort,
+    pub sTypoAscender: libc::c_short,
+    pub sTypoDescender: libc::c_short,
+    pub sTypoLineGap: libc::c_short,
+    pub usWinAscent: libc::c_ushort,
+    pub usWinDescent: libc::c_ushort,
+
+    pub ulCodePageRange1: libc::c_ulong,
+    pub ulCodePageRange2: libc::c_ulong,
+
+    pub sxHeight: libc::c_short,
+    pub sCapHeight: libc::c_short,
+    pub usDefaultChar: libc::c_ushort,
+    pub usBreakChar: libc::c_ushort,
+    pub usMaxContext: libc::c_ushort,
+}
+
+#[repr(C)]
+pub struct TT_Header {
+    pub Table_Version: FT_Fixed,
+    pub Font_Revision: FT_Fixed,
+    pub CheckSum_Adjust: libc::c_long,
+    pub Magic_Number: libc::c_long,
+    pub Flags: libc::c_ushort,
+    pub Units_Per_Em: libc::c_ushort,
+    pub Created: [libc::c_ulong; 2],
+    pub Modified: [libc::c_ulong; 2],
+    pub xMin: libc::c_short,
+    pub yMin: libc::c_short,
+    pub xMax: libc::c_short,
+    pub yMax: libc::c_short,
+    pub Mac_Style: libc::c_ushort,
+    // TODO
+    // FT_UShort  Lowest_Rec_PPEM;
+    //
+    // FT_Short   Font_Direction;
+    // FT_Short   Index_To_Loc_Format;
+    // FT_Short   Glyph_Data_Format;
+}
+
+pub type FT_Face = *mut FT_FaceRec;
+pub type FT_Size = *mut FT_SizeRec;
+pub type FT_CharMap = *mut FT_CharMapRec;
+pub type FT_GlyphSlot = *mut FT_GlyphSlotRec;
+pub type FT_Library = *mut FT_LibraryRec;
+pub type FT_Stream = *mut FT_StreamRec;
+pub type FT_Module = *mut FT_ModuleRec;
+pub type FT_Glyph = *mut FT_GlyphRec;
+pub type FT_Fixed = libc::c_long;
+pub type FT_Error = libc::c_int;
+pub type FT_Pos = libc::c_long;
+pub type FT_String = libc::c_char;
+pub type FT_Generic_Finalizer = extern "C" fn(object: *mut ());
+pub type FT_Pointer = *mut ();
+
+#[link(name = "freetype", kind = "static")]
```

Comment:
It seems like this directive links to a _second_ (static) copy of freetype that doesn't match up with our harfbuzz freetype build. I'm not sure of the consequences, but simply dropping the `#[link]` line both here and in crates/bridge_harfbuzz/src/lib.rs seems to work.

Reply:
Ah, good find, I'll try it locally. I'm using a Windows vcpkg dev environment, and it needs some stuff static so I suspect I added it while narrowing down another issue, and either it's only necessary in some cases, or it's not necessary at all and I just left it in by accident.